cypress/fixtures/dashboard/opensearch_dashboards/visBuilder/prepare_data.md

```diff
@@ -0,0 +1,108 @@
+# Prepare Test data
+
+It may be required to generate the same test data in future with some modifications. In that case, to cause minimal disruption to exsiting tests, follow the following steps.
+
+## Index data
+
+### Generate
+
+To generate test index data, use the generate index data script. The index name is `vis-builder`. The index name is associated directly with the data
+
+```sh
+npm run generate:test-data -- vis-builder \
+  ./cypress/fixtures/dashboard/opensearch_dashboards/visBuilder 
+
+# or
+npm run generate:test-data -- vis-builder \
+  ./cypress/fixtures/dashboard/opensearch_dashboards/visBuilder \
+  -g basic \
+  -n '10000' \
+  -st '2022-01-01T00:00:00.000' \
+  -et '2022-10-01T00:00:00.000'
+```
```

Comment:
this looks nice, but should we add to top level developer guide, so all contribute know and could use it.

Reply:
This doc is specific to vis builder. Its a way to consistently generate the exact same test data in future so that existing tests that rely on this data dont break. The steps that are common to other plugins is in the `writing_tests.md` file